XMPFiles/source/FormatSupport/ID3_Support.cpp

```diff
@@ -871,7 +871,7 @@ void ID3v1Tag::write ( XMP_IO* file, SXMPMeta* meta )
 
 	}
 
-	if ( meta->GetProperty ( kXMP_NS_DM, "trackNumber", &utf8, kXMP_NoOptions ) ) {
+	if ( meta->GetProperty ( kXMP_NS_DM, "trackNumber", &utf8, 0 ) ) {
```

Comment:
Apparently I haven't built master using clang, or at least not 3.8.0 that ships with Fedora 24. It actually generates an error here, and I'd rather use ````nullptr````: correctness over consistency. I will modify the patch accordingly. Thanks !